src/reactapp/package.json

```diff
@@ -1,15 +1,15 @@
 {
-  "name": "hyva-checkout",
-  "version": "0.1.0",
-  "description": "A headless React Checkout for Magento 2",
+  "name": "hyva-react-checkout",
+  "version": "1.0.0",
+  "description": "A React Checkout for Magento 2",
```

Comment:
Good to make the description the same as in the composer. "Highly customizable Magento 2 Checkout, built with React"

Reply:
Updated